App/app.rb

```diff
@@ -0,0 +1,163 @@
+require_relative '../student'
+require_relative '../teacher'
+require_relative '../book'
+require_relative '../rental'
+
+class App
+  def run
+    @option = 0
+    @books = []
+    @people = []
+    @rentals = []
+    show_menu
+  end
+
+  def welcome_message
+    puts 'Welcome to School Library App'
+    puts ''
+  end
+
+  def show_options
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+  end
+
+  def create_student
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp.to_i
+    print 'Has parent permission? [Y/N]: '
+    permission = gets.chomp
+    student = Student.new(age, 0, name, parent_permission: %w[Y y].include?(permission))
+    @people << student
+    puts 'Person creaed successfully'
+    puts ''
+  end
+
+  def create_teacher
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp.to_i
+    print 'Specialization: '
+    specialization = gets.chomp
+    teacher = Teacher.new(age, specialization, name)
```

Comment:
- [ ] When creating a teacher, the data are not being displayed correctly they are misplaced, the name takes the age argument and the age is becoming 0 

![image](https://user-images.githubusercontent.com/75126481/169629306-4a58a52d-b5c4-4aec-8688-caee4ed00ed5.png)

Reply:
- [ ] This should be addressed